posthog/api/user.py

```diff
@@ -124,6 +124,7 @@ def user(request):
                 "ingested_event": team.ingested_event,
             },
             "teams": teams,
+            "has_password": bool(request.user.password),
```

Comment:
Maybe it'd be a bit cleaner to do `has_usable_password` from Django? In case we use some of the other built-in methods